packages/cli/src/commands/generate/scaffold/__tests__/fixtures/components/form.js

```diff
@@ -8,13 +8,11 @@ import {
 } from '@redwoodjs/web'
 
 const CSS = {
-  label: 'block mt-6 text-gray-700 font-semibold',
-  labelError: 'block mt-6 font-semibold text-red-700',
-  input:
-    'block mt-2 w-full p-2 border border-gray-300 text-gray-700 rounded focus:outline-none focus:border-gray-500',
-  inputError:
-    'block mt-2 w-full p-2 border border-red-700 text-red-900 rounded focus:outline-none',
-  errorMessage: 'block mt-1 font-semibold uppercase text-xs text-red-700',
+  label: 'rw-label',
```

Comment:
Now that these are so much simpler, I'd say you should just go ahead and inline the classes. Just get rid of the whole `const CSS` object.

Reply:
Maybe. While I too would personally want these inlined in my projects, I don't agree it's the right decision here. If scaffold generation is meant to ease development and cut down on repetitive tasks, then inlining the classes means a lot of repetitive replacement if you want anything different. Cutting down on repetition is why I initiated this project so I'm inclined to leave it as is. For my personal tastes (inline classes), a quick find and replace will do the trick.

Yeah?

Reply:
With the original code I fully agree. If all the `labelError` classes were inlined on all the labels, then it would have been very repetitive to change the styles of the error messages.

I think the intent of the original code was to let you just change in one place to update the styles for all matching elements. Inlining your semantic classes keeps that intent. You'd only have to change in on place to update the styles of all matching elements - in the css file!

With the semantic classes the point is that you shouldn't have to change the actual classes, right? Only the styles that are applied to the classes.

Reply:
I agree with @Tobbe, I only included `const CSS` because there were so many classes to apply to each element. With all of those gone we can drop the `const` and just have the single CSS class name included right on the element.

Reply:
Fair enough!

I'll tidy that and then this one will be ready for a formal review!